ramanujantools/cmf/cmf.py

```diff
@@ -206,7 +207,7 @@ def trajectory_matrix(self, trajectory: dict, start: dict = None) -> Matrix:
             position[axis] += trajectory[axis]
         if start:
             m = CMF.substitute_trajectory(m, trajectory, start)
-        return m.reduce()
+        return m
 
```

Comment:
This is a great modification since we can now easily do gcd analysis on the potential matrices.

Reply:
Yup, and more importantly - it took ages.